examples/fitness/lib/feed.dart

```diff
@@ -66,19 +66,14 @@ class FeedFragmentState extends State<FeedFragment> {
   void _handleFitnessModeChange(FitnessMode value) {
     setState(() {
       _fitnessMode = value;
-      _drawerShowing = false;
     });
+    config.navigator.pop();
   }
 
-  Drawer buildDrawer() {
-    if (_drawerStatus == AnimationStatus.dismissed)
-      return null;
-    return new Drawer(
-      showing: _drawerShowing,
-      level: 3,
-      onDismissed: _handleDrawerDismissed,
+  void _showDrawer() {
+    showDrawer(
       navigator: config.navigator,
-      children: [
+      child: new Block([
```

Comment:
In some apps, the header doesn't scroll, only the items scroll. What does Material say?